### DISCUSSION TITLE
I want to contribute to foliate, but I don't know how

### DISCUSSION THREAD

**Discussion by mateus-sartorio** [johnfactotum/foliate#1076] [General]
Hello, my name is Mateus, greetings from Brazil. I have some knowledge of JavaScript, and I would really like to contribute to the project. It doesn't have to be with coding, it can be with anything else. I have never contributed to open source projects before, so I would like to know if I could be of service to the project in any way.

**Comment by johnfactotum** [comment]
Thanks. You can look at the issues (especially those labeled *help wanted* or *good first issue*) for a list of things that needs to be done. Note that the GTK 3 version (the master branch) is not being developed anymore, and the GTK 4 version is still in a bit of a state of flux right now. Apart from coding, testing would be appreciated, especially the new renderer used in the GTK 4 version, as it's completely new and less mature than Epub.js and KindleUnpack.
